lib/core/versions/latest/OperationProcessor.ts

```diff
@@ -96,14 +96,17 @@ export default class OperationProcessor implements IOperationProcessor {
       lastOperationTransactionNumber: anchoredOperationModel.transactionNumber
     };
 
+    if (operation.delta === undefined) {
+      return newDidState;
+    }
+
     // Verify the delta hash against the expected delta hash.
-    const deltaPayload = operation.delta ? JsonCanonicalizer.canonicalizeAsBuffer({
+    const deltaPayload = JsonCanonicalizer.canonicalizeAsBuffer({
```

Comment:
Optional opportunistic improvement:

 ```suggestion
    const deltaPayload = JsonCanonicalizer.canonicalizeAsBuffer(operation.delta);
```